tests/kafkatest/services/kafka/templates/kafka.properties

```diff
@@ -25,6 +25,7 @@ process.roles=broker
 {% endif %}
 # The connect string for the controller quorum
 controller.quorum.voters={{ controller_quorum_voters }}
+controller.quorum.bootstrap.servers={{ controller_quorum_bootstrap_servers }}
```

Comment:
do we need an "if this is set, define it, otherwise not" here?

Reply:
I don't believe it's necessary, but I'll do so to make it clear that we may have either or